### PR TITLE
Update Company Value Titles

### PR DESCRIPTION
**Pull request by joepavitt**

### CODE REVIEW

**Comment by joepavitt** [comment]
Really liked the tone of the "Results not Hours" section in the GitLab handbook, and thought we should steal it. Also re-titled some sections to be clearer, and capitalised words as I think it looks nicer.

**Comment by knolleary** [comment]
What's the rationale for changing `Bias for Action` to `Iteration` as they have slightly different meanings?

**Comment by joepavitt** [comment]
> What's the rationale for changing Bias for Action to Iteration as they have slightly different meanings?

I had changed this as found "Bias for Action" quite vague. I took the "Iteration" alternative from the GitLab handbook, and felt it lined up with the main body of text quite well. Won't die on a hill for it though, happy to revert that change.

**Comment by sammachin** [comment]
The most famous place I know Bias for Action from is the Amazon leadership principles https://www.amazon.jobs/en-gb/principles

File: company/values.md [hunk 1]
```diff
@@ -4,20 +4,27 @@
...
+individuals to do what is right. Everyone should feel empowered to do the work
+needed to achieve our goals.
 
 There may be times when more is needed to meet a goal, deadline or customer
 situation. We take the responsibility to ensure they are the exceptions rather
 than the rule.
 
-#### Bias for action
+Do not incite competition by proclaiming how many hours you worked yesterday.
+If you are working too many hours, talk to your manager to discuss solutions.
```
Comment (knolleary) [inline company/values.md:25]: I understand what you're trying to get at - and support the intent - but I'm not sure this is quite the way I would express it. I prefer framing the values as what 'we' do, not what 'you' should (or should not) do.

```suggestion
We don't try to compete over the hours we work - everyone does what is right for them.  We focus on what gets done and take pride in seeing each other succeed.
```

Comment (joepavitt) [reply company/values.md:25]: like that change. The language I'd written was taken straight out of the GitLab handbook, happy to amend.

Comment (ZJvandeWeg) [reply company/values.md:25]: @joepavitt Can you apply that suggestion, and I'll merge this PR after?

Comment (joepavitt) [reply company/values.md]: done

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**